plugins/search-backend-node/api-report.md

```diff
@@ -3,15 +3,61 @@
 > Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
 
 ```ts
-import { DocumentCollator } from '@backstage/search-common';
-import { DocumentDecorator } from '@backstage/search-common';
+/// <reference types="node" />
+
+import { DocumentCollatorFactory } from '@backstage/search-common';
+import { DocumentDecoratorFactory } from '@backstage/search-common';
 import { IndexableDocument } from '@backstage/search-common';
 import { Logger as Logger_2 } from 'winston';
 import { default as lunr_2 } from 'lunr';
 import { QueryTranslator } from '@backstage/search-common';
+import { Readable } from 'stream';
 import { SearchEngine } from '@backstage/search-common';
 import { SearchQuery } from '@backstage/search-common';
 import { SearchResultSet } from '@backstage/search-common';
+import { Transform } from 'stream';
+import { Writable } from 'stream';
+
+// Warning: (ae-missing-release-tag) "BatchSearchEngineIndexer" is exported by the package, but it is missing a release tag (@alpha, @beta, @public, or @internal)
+//
+// @public
+export abstract class BatchSearchEngineIndexer extends Writable {
+  constructor(options: BatchSearchEngineOptions);
+  _final(done: (error?: Error | null) => void): Promise<void>;
```

Comment:
Hmm...  Wonder if `_write()` and `_final()` can/should be marked internal to remove it from the interface?